XRPLib/imu.py

```diff
@@ -110,38 +112,112 @@ def _r_w_reg(self, reg, dat, mask):
         self.rb[0] = (self.rb[0] & mask) | dat
         self._setreg(reg, self.rb[0])
 
-    def _mg(self, reg):
-        return round(self._int16(self._get2reg(reg)) * 0.061 * self._scale_a_c)
+    def is_connected(self):
+        """
+        Checks whether the IMU is connected
+
+        :return: True if WHO_AM_I value is correct, otherwise False
+        :rtype: bool
+        """
+        who_am_i = self._getreg(LSM_REG_WHO_AM_I)
+        return who_am_i == 0x6C
+
+    def reset(self, wait_for_reset = True, wait_timeout_ms = 100):
```

Comment:
Reset should probably stop the update timer and clear all of the offset values

Reply:
Good call! Will add that